.github/actions/release/action.yml

```diff
@@ -13,55 +16,186 @@ inputs:
   jira_project_id:
     description: "Jira project id to create release"
     required: true
-  jira_base_url:
-    description: "Jira base url"
-    required: true
+  jira_webhook_url:
+    description: "Jira release webhook"
 
 runs:
   using: "composite"
   steps:
-    - name: Publish RPMS to Repositories
+    - name: Checkout sources
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Get released versions for components
       run: |
-        NEW_VERSION=""
-        MAJOR_VERSION=$(echo $GITHUB_REF_NAME | grep -oP '([0-9]{2}\.[0-9]{2})')
-        echo "Major version: $MAJOR_VERSION"
-        HOTFIX=$(echo $GITHUB_REF_NAME | grep -oP '(hotfix|)')
-        echo "Hotfix: $HOTFIX"
-        RELEASE_ID=$(git log -1 --pretty=%B | grep -oP '(#[0-9]{4,}#)' | grep -oP '([0-9]+)')
-        echo "Release Id: $RELEASE_ID"
+        set -eu
+        COMPONENTS_RELEASED=()
```

Comment:
is this used at all?

Reply:
not here... IMHO this should be removed

Reply:
indeed this is an old variable that was intedented to hold tags but it was dropped at some point and not used, will be removed